addons/binding/org.openhab.binding.heos/README.md

```diff
@@ -0,0 +1,404 @@
+String HeosKitchen_InputSelect	"Input"		(gHeos)	
+```
+
+Rule for kitchen:
+
+```
+rule "Play AuxIn from Living Room"
+	when
+		Item HeosKitchen_InputSelect received command
+	then
+		if (receivedCommand.toString == "aux_in_1") {
+			sendCommand(HeosKitchen_Input, "aux_in_1")
+			
+		} if (receivedCommand.toString == "LivingRoom") {
+			sendCommand(HeosBridge_Play_Living, ON)
+			sendCommand(HeosKitchen_Input, "aux_in_1")
+			sendCommand(HeosBridge_Play_Living, OFF)	//Switch player channel off again to be sure that it is OFF
+		}
+```
+
+Sitemap:
+
+```
+Switch item=HeosKitchen_InputSelect	mappings=[aux_in_1 = "Aux In" , LivingRoom = "Living Room"]
+```
+
+### The OnlineState of Groups and Players
+
+The online state of a Thing can be helpful for groups to control the visibility of group items within sitemap. So if the group is removed the visibility of those items is also changed.
+
+#### Example
+
+First you have to define a new Item within the Item section which is used later within the Sitemap:
+
+Items:
+
+```
+String HeosGroup_Status
+
+```
+
+Then we need a rule which triggers the state if an Item goes Online or Offline.
+
+Rules:
+
+```
+rule "Online State Heos Group" 
+
+when
+    Thing "heos:group:1747557118" changed 
+then
+    var thingStatus = getThingStatusInfo("heos:group:1747557118")
+    sendCommand(HeosGroup_Status, thingStatus.getStatus.toString)    
+end
+
+```
+
+Sitemap:
+
+```
+Frame label="Heos Group" visibility=[HeosGroup_Status==ONLINE] {
+	
+	Default item=HeosGroup1_Player
+	Default item=HeosGroup1_Volume
+	Default item=HeosGroup1_Mute
+	Default item=HeosGroup1_Favorites
+	Default item=HeosGroup1_Playlist
+	
+	Text item=HeosGroup1_Song {
+		Default item=HeosGroup1_Song
+		Default item=HeosGroup1_Artist
+		Default item=HeosGroup1_Album
+		Image item=HeosGroup1_Cover url=""
+	}
+	
+}
+```
+
+### Dynamic Group Handling (Keep Groups)
+
+Groups are a more or less dynamic item compared to the players. The players are always present within the network and are only added or removed if they are added or removed physically. Groups on the other hand are dynamic because they are only virtual created and removed. 
+Therefore dynamic group handling only sets groups to OFFLINE if the option is activated. This is indicated via the "OnlineStatus" channel of the group. If the group then is created again the binding automatically sets the status to online without the need of action by the user.
+If dynamic group handling is deactivated the group thing is completely removed if the binding is notified about a removed group. The group then has to be added manually again if created.
+
+
+### Playlists
+
+Playlists can be played by sending the number (starts at 0!) to the binding via the playlists channel at the corresponding player or group. To find the correct number for the playlist, please have a look to the HEOS App and see at which position the playlist you want to play is located.
```

Comment:
I really like how it works now!

Maybe it would suite to apply the same logic for the favorites :-)

Reply:
Good to hear!
You mean instead of having a dedicated channels for each favorite?

Reply:
No to move it from the bridge to the device / group.

Reply:
> You mean instead of having a dedicated channels for each favorite?

Although that would maybe also work well...

Reply:
I started to have a look on it, but these requires a little bit more work, because I have to change how the channels are created. 
But I will continue working on it.